src/models/partial_tx.ts

```diff
@@ -596,7 +596,7 @@ export class PartialTxInputData {
    */
   addSignatures(serialized: string) {
     const arr = serialized.split('|');
-    if (arr.length < 3 || arr[0] != 'PartialTxInputData' || arr[1] !== this.hash) {
+    if (arr.length < 2 || arr[0] != 'PartialTxInputData' || arr[1] !== this.hash) {
```

Comment:
Two comments about this line:

Why did you change the amount of elements in this `serialized` array? Is this related to a specific change?

And second, I think it would be better to use the string constant already there:
```suggestion
    if (arr.length < 2 || arr[0] != PartialTxInputDataPrefix || arr[1] !== this.hash) {
```